packages/lzards-api-client/package.json

```diff
@@ -16,7 +16,7 @@
     "coverage": "python ../../scripts/coverage_handler/coverage.py"
   },
   "ava": {
-    "fail-fast": true,
```

Comment:
oops 😓

Reply:
This was actually the previous syntax. The ava update changed it to camel case.